technote.toml

```diff
@@ -0,0 +1,9 @@
+[technote]
```

Comment:
@craiglagegit the author list is missing. Can you add it in please?

I think you need to run `make add-author`.